### DISCUSSION TITLE
Changed subnet & HA IP, Frigate won’t turn back on.

### DISCUSSION THREAD

**Discussion by Normanras** [blakeblackshear/frigate#3459] [Ask A Question]
Title sort of says it all but I changed my home's subnet since my VPN was interfering with other networks with the same subnet. 

I updated HA's static and restarted along with restarting everything else in the network. I then updated my frigate and HA yaml files for my cameras to the new IP (and confirmed they are correct with the correct username etc), restarted again. 

I even tried paring down my frigate.yml to barebones - and nothing. 

Frigate won't turn on. I tried removing it from HACS and installing it again from scratch, but I can't figure it out. The logs show nothing and in integrations it (obviously) says it can't connect. 

This seems awfully specific, but anyone experience something similar?

**Comment by NickM-27** [comment]
The addon should surely show something in the logs? Also what is your config? Would suggest making sure the connection to MQTT is working as expected
